internal/scene/play.go

```diff
@@ -43,6 +44,7 @@ func NewPlay(c controller) *Play {
 	p.registerAction(ebiten.KeyT, action.ToggleTextures)
 	p.registerAction(ebiten.KeyC, action.ToggleCollisions)
 	p.registerAction(ebiten.KeyD, action.ToggleCollisionBoxes)
+	p.registerAction(ebiten.Key(ebiten.MouseButtonLeft), action.DropFood)
```

Comment:
I'm not sure if this would work. I think the Key and MouseButton are two different types. I'll look into it.